yazi-adapter/src/emulator.rs

```diff
@@ -67,6 +69,7 @@ impl Emulator {
 			("WT_Session", Self::Microsoft),
 			("VSCODE_INJECTION", Self::VSCode),
 			("TABBY_CONFIG_DIRECTORY", Self::Tabby),
+			("TERMINAL_NAME", Self::Contour),
```

Comment:
This can't be used as a "special" env variable as it's not "special"

Reply:
I deleted it.